app/controllers/pdfs_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PdfsController < ApplicationController
+  def show
+    pdf = Pdf.find(params[:id])
+    send_file pdf.pdf_file.url, :type=>"application/pdf", :x_sendfile=>true
```

Comment:
This is delivering the download of only the file locally. I will need to make this work through s3 eventually.